src/components/ChartContainer/ChartBarDateRange.js

```diff
@@ -1,9 +1,11 @@
-import React from 'react';
-
-function ChartBarDateRange() {
-    return (
-        
-    );
-}
-
-export default ChartBarDateRange;
+import React from 'react';
+
+function ChartBarDateRange() {
```

Comment:
I don't see the purpose or use of this class anywhere.

Reply:
delete it